src/scss/3_components/form/form.scss

```diff
@@ -0,0 +1,111 @@
+/*
+ * @file
+ * Provides the generic styling for all types of form components.
+ */
+@import '../../init';
+
+fieldset,
+.fieldset {
+  border: 0;
+
+  &:focus {
+    outline: 0;
+    border: 0;
+  }
+}
+
+select,
+[type='text'],
+[type='tel'],
+[type='url'],
+[type='email'],
+[type='password'],
+[type='date'],
+[type='time'],
+[type='number'],
+//input:not([type='button']):not([type='submit']):not([type='radio']):not([type='checkbox']):not([type='search']),
```

Comment:
Remove commented out.

Reply:
haven't fully finished with this file, I'll remove - it was left to remind me what the original declaration was.